src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -469,6 +470,32 @@ public function register(Application $app)
 
                 $listener->addHandler(new SessionLogoutHandler());
 
+                $removeCookies = isset($options['remove_cookies']) ? $options['remove_cookies'] : null;
+                if (null !== $removeCookies) {
+                    $cookies = array();
+
+                    foreach ($removeCookies as $key => $value) {
+                        if (is_array($value)) {
+                            // Check for existence of method parameters otherwise
+                            // use default values given by ResponseHeaderBag:clearCookie()
+                            if (!isset($value['path'])) {
+                                $value['path'] = '/';
```

Comment:
It would be nice if the `CookieClearingLogoutHandler` could make this check itself.